inst/examples/AA-AA-000-0000/workflows/2_metrics/kri0001.yaml

```diff
@@ -0,0 +1,77 @@
+meta:
+  Type: Analysis
+  ID: kri0001
+  File: kri0001.yaml
+  GroupLevel: Site
+  Abbreviation: AE
+  Metric: Adverse Event Rate
+  Numerator: Adverse Events
+  Denominator: Days on Study
+  Model: Normal Approximation
+  Score: Adjusted Z-Score
+  AnalysisType: rate
+  Threshold: -2,-1,2,3
+  nMinDenominator: 30
+spec:
+  Mapped_AE:
+    subjid:
+      required: true
+      type: character
+  Mapped_SUBJ:
+    subjid:
+      required: true
+      type: character
+    invid:
+      required: true
+      type: character
+    timeonstudy:
+      required: true
+      type: numeric
+steps:
+  - name: ParseThreshold
+    output: vThreshold
+    params:
+      strThreshold: Threshold
+  - name: Input_Rate
+    output: Analysis_Input
+    save: true
+    params:
+      dfSubjects: Mapped_SUBJ
+      dfNumerator: Mapped_AE
+      dfDenominator: Mapped_SUBJ
+      strSubjectCol: subjid
+      strGroupCol: invid
+      strGroupLevel: GroupLevel
+      strNumeratorMethod: Count
+      strDenominatorMethod: Sum
+      strDenominatorCol: timeonstudy
+  - name: Transform_Rate
+    output: Analysis_Transformed
+    params:
+      dfInput: Analysis_Input
+  - name: Analyze_NormalApprox
+    output: Analysis_Analyzed
+    params:
+      dfTransformed: Analysis_Transformed
+      strType: AnalysisType
+  - name: Flag_NormalApprox
+    output: Analysis_Flagged
+    params:
+      dfAnalyzed: Analysis_Analyzed
+      vThreshold: vThreshold
+  - name: Summarize
+    output: Analysis_Summary
+    save: true
+    params:
+      dfFlagged: Analysis_Flagged
+      nMinDenominator: nMinDenominator
+  - name: list
```

Comment:
though I suppose its up to the user at the end of the day whether or not add this list object, was it intentional to keep it here and remove from `inst/workflow`?

Reply:
oh nvm don't mind me, didn't read the above comments

Reply:
The way we are going to "finalize" the yamls have downstream impact on #1862